daemon/create_unix.go

```diff
@@ -62,7 +60,7 @@ func (daemon *Daemon) createContainerOSSpecificSettings(container *container.Con
 			return fmt.Errorf("cannot mount volume over existing file, file exists %s", path)
 		}
 
-		v, err := daemon.volumes.Create(context.TODO(), name, hostConfig.VolumeDriver, volumeopts.WithCreateReference(container.ID))
+		v, err := daemon.volumes.Create(context.TODO(), "", hostConfig.VolumeDriver, volumeopts.WithCreateReference(container.ID))
```

Comment:
For a follow-up; given that name is optional, we should change this signature;

- move name to options
- and/or require "anonymous" to be set as option (if we need to disambiguate "forgot to pass a name" from "must be anonymous, and generate a name for me")

https://github.com/moby/moby/blob/2fa66cfce2523f6e38bcccc1161055b46c3c6a90/volume/service/service.go#L72-L76